05/app.js

```diff
@@ -1,53 +1,86 @@
 const stats = {
-    paragraphs: {
-        'p1': 0,
-    },
-    links: {
-        '/dolor.html': 0,
-    }
+  paragraphs: {
+    p1: 0,
+  },
+  links: {
+    "/dolor.html": 0,
+  },
 };
 
 /* tutaj umieść swój kod */
 
+const paragraphs = document.querySelectorAll("p");
+paragraphs.forEach(function (el) {
+  el.addEventListener("click", function (e) {
+    tag = e.target.tagName;
+    id = e.target.dataset.id;
+    href = e.target.getAttribute("href");
```

Comment:
Brak deklaracji tj. `let`

PS. Raczej pobierałbym dane dopiero wtedy jak wiem jakie potrzebuje, a nie wszystkie (w przypadku większe ilości danych byłoby to nie optymalne rozwiązanie)